.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,69 @@
+name: Bug Report
+description: File a bug report.
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: "**Required Info:**"
+  - type: input
+    id: os
+    attributes:
+      label: "Operating System:"
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: "Version or commit hash:"
+    validations:
+      required: true
+  - type: input      
+    id: dds
+    attributes:
+      label: "DDS implementation:"
```

DDS implementation is the one of the RMW implementation, this should be `RMW` (ROS Middle Ware).

```suggestion
      label: "RMW implementation:"
```